app/views/pages/using_registers.html.haml

```diff
@@ -39,6 +39,6 @@
         %li download either the whole or part of a register
         %li find a single record, entry or item
 
-      %p The #{link_to 'technical documentation', 'https://registers-docs-testing.cloudapps.digital', target: :blank, rel: 'noopener noreferrer'} has more information.
+      %p The #{link_to 'technical documentation', 'https://docs.registers.service.gov.uk', target: :blank, rel: 'noopener noreferrer'} has more information.
```

Comment:
why `noreferrer` surely this is useful information to know how people are getting to the docs?

Reply:
We don't need this :)